packages/stark-core/src/logging/services/logging.service.spec.ts

```diff
@@ -181,7 +182,34 @@ describe("Service: StarkLoggingService", () => {
 			expect((<StarkLogMessage>message).message).toContain("end of message");
 			expect((<StarkLogMessage>message).timestamp).toBeDefined();
 			expect((<StarkLogMessage>message).error).toBeDefined();
-			expect((<StarkLogMessage>message).error).toContain("this is the error");
+
+			const error: StarkError = (<StarkError>(<StarkLogMessage>message).error);
+			if (error) {
+				expect((<string>(error).message)).toContain("this is the error");
+			}
+		});
+
+		it("should dispatch LOG_MESSAGE action when no Error object is provided", () => {
+			const errorMsg: string = "dummy error message";
+			loggingService.error(errorMsg);	// pass only a message
+			expect(mockStore.dispatch).toHaveBeenCalledTimes(1);
+			const dispatchedAction: Action = (<Spy>mockStore.dispatch).calls.mostRecent().args[0];
+			expect(dispatchedAction.type).toBe(StarkLoggingActionTypes.LOG_MESSAGE);
+			const { message }: any = dispatchedAction;
+			expect(message instanceof StarkLogMessageImpl).toBe(true);
+			expect((<StarkLogMessage>message).type).toBe(StarkLogMessageType.ERROR);
+			expect((<StarkLogMessage>message).message).toContain("dummy error message");
+			expect((<StarkLogMessage>message).timestamp).toBeDefined();
+			expect((<StarkLogMessage>message).error).toBeDefined();
+
+			const error: StarkError = (<StarkError>(<StarkLogMessage>message).error);
+			if (error) {
```

Comment:
Same here, we could replace the if block with an expectation (see my previous comment)